docs/nextmv/tutorials/applications/push.md

```diff
@@ -10,23 +10,34 @@ following minimum requirements:
 
 1. Must read from either `stdin` or a file.
 2. Must write results either to `stdout` or a file.
-3. Must contain an [`app.yaml` manifest][app-manifest] in the root of your app
-project.
+3. Depending on the push strategy (see below), may require an `app.yaml`
```

Comment:
Sorry for dragging this further, but really the app.yaml is non-negotiable, so it is not correct that an app "may" need a manifest. It always does. But it is ok.